index.html

```diff
@@ -265,14 +265,32 @@ <h3>JsonSchemaCredential</h3>
                <b>JsonSchemaCredential</b> is used for the validation of W3C Verifiable Credentials using
                JSON Schema, where the JSON Schema is contained with a <a>verifiable credential</a>.
                When dereferencing the <code>id</code> property associated with the
-               <code>JsonSchemaCredential</code> <code>type</code> value the result is a valid 
-               <a>verifiable credential</a>. The resulting <a>verifiable credential</a>'s  
-               <code>credentialSubject</code> property MUST contain a two properties:
-               <ul>
+               <code>credentialSchema</code> <code>type</code> value, the result is a valid
+               <a>verifiable credential</a>. For the resulting <a>verifiable credential</a>:
+            <p>
+            <ul>
+              <li>
+                The <code>credentialSubject</code> property MUST contain two properties:
+                <ul>
                   <li><code>type</code> – the value of which MUST be <code>JsonSchema</code></li>
                   <li><code>jsonSchema</code> – an object which contains a valid JSON Schema</li>
-               </ul>
-            </p>
+                </ul>
+              </li>
+              <li>
+                The value of the <code>credentialSchema</code> property MUST always be set to:
+                <pre title="Value of a JsonSchemaCredential's credentialSchema property">
+                {
+                  "id": "https://www.w3.org/2022/credentials/v2/json-schema-credential-schema.json",
```

Comment:
This is confusing to me... it sounds like the JsonSchemaCredential can only have one schema associated with it?

Reply:
> it sounds like the JsonSchemaCredential can only have one schema associated with it?

Yes.

> Doesn't this result in a ton of repetition in each VC?

Yes, it will be repeated for all VCs who's `type` property has the value `JsonSchemaCredential`. This is no different than specifying a `credentialSchema` property on any other VC.

Reply:
I updated the example, which I had originally missed. I'm hoping this clarifies things further.

Reply:
@msporny this is a 'meta schema' 

The idea is to prevent a credential schema having a credential schema having a credential schema....

So this limits the nesting to a single level:
- A JsonSchemaCredential can have a single JsonSchema
- A JsonSchema can have a single jsonSchema